integration-tests/stress_test.go

```diff
@@ -0,0 +1,172 @@
+package integration_tests
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/datastax/zdm-proxy/integration-tests/env"
+	"github.com/datastax/zdm-proxy/integration-tests/setup"
+	"github.com/gocql/gocql"
+	"github.com/rs/zerolog"
+	log "github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"sync"
+	"sync/atomic"
+	"testing"
+	"time"
+)
+
+func TestSimultaneousConnections(t *testing.T) {
+	if !env.RunCcmTests {
+		t.Skip("Test requires CCM, set USE_CCM env variable to TRUE")
```

Comment:
This message should say `Test requires CCM, set RUN_CCMTESTS env variable to TRUE`.